python/dgl/distributed/dist_context.py

```diff
@@ -138,7 +154,14 @@ def is_initialized():
     return INITIALIZED
 
 def exit_client():
-    """Register exit callback.
+    """Trainer exits
+
+    This function is called automatically when a Python process exits. Normally,
+    the training script does not need to invoke this function at the end.
+
+    In the case that the training script needs to initialize the distributed module
+    multiple times (so far, this is needed in the unit tests), the training script
+    needs to call `exit_client` before calling `initialize` again.
```

Comment:
Can we just hide this api?

Reply:
this is docstring. it's not listed in the rst file.